src/System.Memory/tests/ParsersAndFormatters/Parser/ParserTests.2gbOverflow.cs

```diff
@@ -0,0 +1,108 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Text;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Runtime.CompilerServices;
+
+using Xunit;
+
+namespace System.Buffers.Text.Tests
+{
+    public static partial class ParserTests
+    {
+        private const int TwoGiB = int.MaxValue;
+
+        [Fact]
+        [OuterLoop]
+        public static void TestParser2GiBOverflow()
```

Comment:
Only run such tests on Windows/OSX. See https://github.com/dotnet/corefx/blob/master/src/System.Memory/tests/Span/Clear.cs#L236

Reply:
As it stands right now, it's not specific to Windows...

Reply:
I was suggesting excluding running this test on Linux. We have seen an issue in the past with trying to use OOM exception to skip allocation on Linux:
> On Linux, the allocation can succeed even if there is not enough memory but then the test may get killed by the OOM killer at the time the memory is accessed which triggers the full memory allocation.

Reply:
Excluded